### ISSUE TITLE
WC CLI product_attribute_term create has no attribute_id option

### ISSUE THREAD

**Issue by theMikeD** [woocommerce/woocommerce#13126]
## EXPLANATION OF THE ISSUE
Perhaps this is a conceptual problem. I have two attributes "foo" and "bar" each with their own set of terms. What I would like to do is add a new term to "foo" only...yet the CLI command to create terms doesn't include a `--parent` or `--attribute` option. So how do I add a term to a specified attribute if I can't specify that attribute?

The API docs here http://woocommerce.github.io/woocommerce-rest-api-docs/#create-an-attribute-term specify an <attribute_id> in the path to create a term...but it's not in the CLI docs and using `--attribute` or `--product_attribute` generates an error. Is it in fact there and just not listed?

Note that I am not talking about product-specific attribute terms here. I'm talking about the terms shown when you go to Products->Attributes->Configure Terms

I'm also getting a 404 when I try to do anything with product_attribute_term, maybe that is related?

## STEPS TO REPRODUCE THE ISSUE

```
[mike@macster: /WOO/] $ wp --info
PHP binary:	/Applications/MAMP/bin/php/php7.1.0/bin/php
PHP version:	7.1.0
php.ini used:	/Applications/MAMP/bin/php/php7.1.0/conf/php.ini
WP-CLI root dir:	phar://wp-cli.phar
WP-CLI packages dir:	
WP-CLI global config:	
WP-CLI project config:	
WP-CLI version:	1.1.0

[mike@macster: /WOO/] $ wp wc product_attribute_term
usage: wp wc product_attribute_term create --name=<name> [--slug=<slug>] [--description=<description>] [--menu_order=<menu_order>] [--porcelain]
   or: wp wc product_attribute_term delete <id> [--force=<force>] [--porcelain]
   or: wp wc product_attribute_term get <id> [--context=<context>] [--fields=<fields>] [--field=<field>] [--format=<format>]
   or: wp wc product_attribute_term list [--context=<context>] [--page=<page>] [--per_page=<per_page>] [--search=<search>] [--exclude=<exclude>] [--include=<include>] [--order=<order>] [--orderby=<orderby>] [--hide_empty=<hide_empty>] [--parent=<parent>] [--product=<product>] [--slug=<slug>] [--fields=<fields>] [--field=<field>] [--format=<format>]
   or: wp wc product_attribute_term update <id> [--name=<name>] [--slug=<slug>] [--description=<description>] [--menu_order=<menu_order>] [--porcelain]

See 'wp help wc product_attribute_term <command>' for more information on a specific command.

[mike@macster: /WOO/] $ wp wc product_attribute_term create --name='test'--user=miked --debug=wc
Debug: No schema title found for /wc/v1, skipping REST command registration. (0.4s)
Notice: Undefined offset: 0 in /Volumes/EVO/Active_Sites/PP/TESTING/wp-content/plugins/woocommerce/includes/cli/class-wc-cli-rest-command.php on line 329
Debug: wc command executed 0 queries in 0 seconds (0.428s)
Error: No route was found matching the URL and request method {"status":404}
```

## SYSTEM STATUS REPORT

```
### WordPress Environment ###

Home URL: http://woo.dev
Site URL: http://woo.dev/wordpress
WC Version: 2.7.0
Log Directory Writable: ✔
WP Version: 4.7.2
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: –
Language: en_CA

### Server Environment ###

Server Info: Apache
PHP Version: 7.1.0
PHP Post Max Size: 32 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.51.0
OpenSSL/1.0.2j

SUHOSIN Installed: –
MySQL Version: 5.6.34
Max Upload Size: 32 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 2.7.0
WC Database Prefix: wp_
woocommerce_sessions: ✔
woocommerce_api_keys: ✔
woocommerce_attribute_taxonomies: ✔
woocommerce_downloadable_product_permissions: ✔
woocommerce_order_items: ✔
woocommerce_order_itemmeta: ✔
woocommerce_tax_rates: ✔
woocommerce_tax_rate_locations: ✔
woocommerce_shipping_zones: ✔
woocommerce_shipping_zone_locations: ✔
woocommerce_shipping_zone_methods: ✔
woocommerce_payment_tokens: ✔
woocommerce_payment_tokenmeta: ✔
MaxMind GeoIP Database: ✔

### Security ###

Secure connection (HTTPS): ❌Your store is not using HTTPS. Learn more about HTTPS and SSL Certificates.
Hide errors from visitors: ✔

### Active Plugins (9) ###

Admin Post Navigation: by Scott Reilly – 2.0
Transient Cleaner: by David Artiss – 1.5.2
MikeD's De-Bugging Helper: by MikeD – 1.0
Password Protected: by Ben Huson – 2.0.3
Soliloquy - WooCommerce Addon: by Soliloquy Team – 1.1.3
Soliloquy: by Soliloquy Team – 2.5.3.1
WooCommerce Product CSV Import Suite: by Automattic – 1.10.12
WooCommerce: by WooThemes – 2.7.0-beta-1

### Settings ###

API Enabled: ✔
Force SSL: –
Currency: CAD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)


### WC Pages ###

Shop base: #3 - /shop/
Cart: #4 - /cart/
Checkout: #5 - /checkout/
My account: #6 - /my-account/

### Theme ###

Name: Jessica
Version: 1.3.1
Author URL: http://www.web-savvy-marketing.com/
Child Theme: ✔
Parent Theme Name: Genesis
Parent Theme Version: 2.4.2
Parent Theme Author URL: https://www.studiopress.com/
WooCommerce Support: ✔

### Templates ###

Overrides: jessica/taxonomy-product_tag.php
````

**Comment by theMikeD** [comment]
So if I use the master branch this should be fixed via `--id` is that right? There should be a bug report to update the docs (web and built-in) too right?

**Comment by justinshreve** [comment]
The built-in docs are generated from the code. The web/wiki needs updated, which I can do tomorrow.

**Comment by theMikeD** [comment]
```
[mike@macster: /WOO/] $ wp wc product_attribute_term
usage: wp wc product_attribute_term create <attribute_id> [--attribute_id=<attribute_id>] --name=<name> [--slug=<slug>] [--description=<description>] [--menu_order=<menu_order>] [--porcelain]
   or: wp wc product_attribute_term delete <attribute_id> <id> [--id=<id>] [--attribute_id=<attribute_id>] [--force=<force>] [--porcelain]
   or: wp wc product_attribute_term get <attribute_id> <id> [--id=<id>] [--attribute_id=<attribute_id>] [--context=<context>] [--fields=<fields>] [--field=<field>] [--format=<format>]
   or: wp wc product_attribute_term list <attribute_id> [--attribute_id=<attribute_id>] [--context=<context>] [--page=<page>] [--per_page=<per_page>] [--search=<search>] [--exclude=<exclude>] [--include=<include>] [--order=<order>] [--orderby=<orderby>] [--hide_empty=<hide_empty>] [--parent=<parent>] [--product=<product>] [--slug=<slug>] [--fields=<fields>] [--field=<field>] [--format=<format>]
   or: wp wc product_attribute_term update <attribute_id> <id> [--id=<id>] [--attribute_id=<attribute_id>] [--name=<name>] [--slug=<slug>] [--description=<description>] [--menu_order=<menu_order>] [--porcelain]

See 'wp help wc product_attribute_term <command>' for more information on a specific command.
```


`<attribute_id> [--attribute_id=<attribute_id>]` ?

**Comment by theMikeD** [comment]
I can confirm that the list and create commands now work as expected

**Comment by justinshreve** [comment]
Hmm - I'm not seeing that ith the built in docs:

```
$ wp wc product_attribute_term
usage: wp wc product_attribute_term create <attribute_id> --name=<name> [--slug=<slug>] [--description=<description>] [--menu_order=<menu_order>] [--porcelain]
   or: wp wc product_attribute_term delete <attribute_id> <id> [--force=<force>] [--porcelain]
   or: wp wc product_attribute_term get <attribute_id> <id> [--context=<context>] [--fields=<fields>] [--field=<field>] [--format=<format>]
   or: wp wc product_attribute_term list <attribute_id> [--context=<context>] [--page=<page>] [--per_page=<per_page>] [--search=<search>] [--exclude=<exclude>] [--include=<include>] [--order=<order>] [--orderby=<orderby>] [--hide_empty=<hide_empty>] [--parent=<parent>] [--product=<product>] [--slug=<slug>] [--fields=<fields>] [--field=<field>] [--format=<format>]
   or: wp wc product_attribute_term update <attribute_id> <id> [--name=<name>] [--slug=<slug>] [--description=<description>] [--menu_order=<menu_order>] [--porcelain]

See 'wp help wc product_attribute_term <command>' for more information on a specific command.
```

Are you still?

**Comment by theMikeD** [comment]
Yes I am. #feeb98e80.

```
[mike@macster: /WOO/] $ wp wc product_attribute_term
usage: wp wc product_attribute_term create <attribute_id> [--attribute_id=<attribute_id>] --name=<name> [--slug=<slug>] [--description=<description>] [--menu_order=<menu_order>] [--porcelain]
   or: wp wc product_attribute_term delete <attribute_id> <id> [--id=<id>] [--attribute_id=<attribute_id>] [--force=<force>] [--porcelain]
   or: wp wc product_attribute_term get <attribute_id> <id> [--id=<id>] [--attribute_id=<attribute_id>] [--context=<context>] [--fields=<fields>] [--field=<field>] [--format=<format>]
   or: wp wc product_attribute_term list <attribute_id> [--attribute_id=<attribute_id>] [--context=<context>] [--page=<page>] [--per_page=<per_page>] [--search=<search>] [--exclude=<exclude>] [--include=<include>] [--order=<order>] [--orderby=<orderby>] [--hide_empty=<hide_empty>] [--parent=<parent>] [--product=<product>] [--slug=<slug>] [--fields=<fields>] [--field=<field>] [--format=<format>]
   or: wp wc product_attribute_term update <attribute_id> <id> [--id=<id>] [--attribute_id=<attribute_id>] [--name=<name>] [--slug=<slug>] [--description=<description>] [--menu_order=<menu_order>] [--porcelain]

See 'wp help wc product_attribute_term <command>' for more information on a specific command.
[mike@macster: /WOO/] $ wp --info
PHP binary:	/Applications/MAMP/bin/php/php7.1.0/bin/php
PHP version:	7.1.0
php.ini used:	/Applications/MAMP/bin/php/php7.1.0/conf/php.ini
WP-CLI root dir:	phar://wp-cli.phar
WP-CLI packages dir:	
WP-CLI global config:	
WP-CLI project config:	
WP-CLI version:	1.1.0
[mike@macster: /WOO/] $ 
```

**Comment by justinshreve** [comment]
Do you see this with other commands that take multiple IDs? For example `wc shop_order_refund`?

**Comment by justinshreve** [comment]
The CLI wiki docs have been updated.

**Comment by theMikeD** [comment]
```
[mike@macster: /WOO/] $ wp wc shop_order_refund
usage: wp wc shop_order_refund create <order_id> [--order_id=<order_id>] [--amount=<amount>] [--reason=<reason>] [--refunded_by=<refunded_by>] [--meta_data=<meta_data>] [--line_items=<line_items>] [--porcelain]
   or: wp wc shop_order_refund delete <order_id> <id> [--order_id=<order_id>] [--id=<id>] [--force=<force>] [--porcelain]
   or: wp wc shop_order_refund get <order_id> <id> [--order_id=<order_id>] [--id=<id>] [--context=<context>] [--fields=<fields>] [--field=<field>] [--format=<format>]
   or: wp wc shop_order_refund list <order_id> [--order_id=<order_id>] [--context=<context>] [--page=<page>] [--per_page=<per_page>] [--search=<search>] [--after=<after>] [--before=<before>] [--exclude=<exclude>] [--include=<include>] [--offset=<offset>] [--order=<order>] [--orderby=<orderby>] [--filter=<filter>] [--status=<status>] [--customer=<customer>] [--product=<product>] [--dp=<dp>] [--fields=<fields>] [--field=<field>] [--format=<format>]

See 'wp help wc shop_order_refund <command>' for more information on a specific command.
[mike@macster: /WOO/] $ 
```

**Comment by theMikeD** [comment]
Fixed in #13145 from what I can see

```
[mike@macster: /woocommerce/] $ git co fix/cli-id-docs
Branch fix/cli-id-docs set up to track remote branch fix/cli-id-docs from origin.
Switched to a new branch 'fix/cli-id-docs'
```
```
[mike@macster: /WOO/] $ wp wc product_attribute_term
usage: wp wc product_attribute_term create <attribute_id> --name=<name> [--slug=<slug>] [--description=<description>] [--menu_order=<menu_order>] [--porcelain]
   or: wp wc product_attribute_term delete <attribute_id> <id> [--force=<force>] [--porcelain]
   or: wp wc product_attribute_term get <attribute_id> <id> [--context=<context>] [--fields=<fields>] [--field=<field>] [--format=<format>]
   or: wp wc product_attribute_term list <attribute_id> [--context=<context>] [--page=<page>] [--per_page=<per_page>] [--search=<search>] [--exclude=<exclude>] [--include=<include>] [--order=<order>] [--orderby=<orderby>] [--hide_empty=<hide_empty>] [--parent=<parent>] [--product=<product>] [--slug=<slug>] [--fields=<fields>] [--field=<field>] [--format=<format>]
   or: wp wc product_attribute_term update <attribute_id> <id> [--name=<name>] [--slug=<slug>] [--description=<description>] [--menu_order=<menu_order>] [--porcelain]

See 'wp help wc product_attribute_term <command>' for more information on a specific command.
```
